statsdbinterface/function_cache.py

```diff
@@ -1,6 +1,12 @@
 import time
 from threading import Thread, Lock
 import atexit
+try:
+    import config
+    cache_enabled = config.CACHE
```

Comment:
Just see this now. That is really bad design in terms of Flask. You should **NEVER** import a configuration module like that. Use the current `app.config` or dynamically query `current_app.config` during requests. The reason for this is a design decision made in the Flask framework, and even though it might work, the Flask way is the one to choose.